### DISCUSSION TITLE
Customize appearance of focus indicator ("my focus and: [ ]")

### DISCUSSION THREAD

**Discussion by adigitoleo** [zellij-org/zellij#3534] [Q&A]
When I connect to a remote machine with a running zellij session and attach to the same session, then the fact that multiple machines are viewing the same session is helpfully indicated by:
- a colored block cursor symbol in the pane being viewed by the other machine(s)
- the header of the current pane border which either prints "my focus" or "my focus and: " followed by the colored block cursor symbol(s)

It's great that this information is presented in an obvious way. However, I wonder if there is a way to customize the indicator, e.g. instead of colored block cursors use ~~host names~~, or ascii symbols?

Edit: realised that using hostnames is potentially non-unique.

Also, I've noticed that when I connect to the same remote machine/zellij session with two separate terminals, the pane border drawing seems to get out of whack: either the border disappears, or only the colored cursors appear, or some similar minor visual corruption occurs. I was using 0.40.1 and connecting from Win11 to an Ubuntu machine. Is this a known issue/not supported or should I try to report this properly if I get time. I don't usually run duplicate connections in that way so it's not a big deal.

**Comment by Lockszmith-GH** [comment]
For me (Also using Windows Terminal from Win11) the only artifact I have for mulitple terminal connections is the size of the broder is reduced to the smallest terminal window - which is the wanted effect.

**Reply by adigitoleo** [reply]
Thanks for checking. I do see this behaviour as well, and that makes sense. I'll see if I can dig deeper into why I'm getting the other visual artifacts at some point.

**Comment by tracure1337** [comment]
I am also interested in being able to edit the text.

**Comment by mwinters0** [comment]
My laptop crashed, and when I re-attached I'm now stuck with MY FOCUS AND [BRIGHT PINK SQUARE].  Super distracting.  If there's a way to boot my old attachment, I don't see it.

**Reply by imsnif** [reply]
You can do this through the session-manager (`Ctrl o` + `w`) by pressing `Ctrl x` (should also appear in the help-line of same).
